OtripleS.Web.Api.Tests.Unit/Services/StudentServiceTests/StudentServiceTests.cs

```diff
@@ -36,5 +37,17 @@ private Student CreateRandomStudent()
             
             return filler.Create();
         }
+
+        private IQueryable<Student> CreateRandomStudents()
+        {
+            int randomNumber = new IntRange(min: 2, max: 10).GetValue();
+            var filler = new Filler<Student>();
+            filler.Setup()
+               .OnProperty(student => student.BirthDate).Use(this.dateTimeBroker.GetCurrentDateTime())
+               .OnProperty(student => student.CreatedDate).Use(this.dateTimeBroker.GetCurrentDateTime())
```

Comment:
Separate multi-liner statements from one-liners with a new empty line.
Here's some guidlines around the topic:
https://github.com/hassanhabib/CSharpCodingStandard/blob/master/Variables.md#021-multiple-declarations